Contents/Sketch/projects.js

```diff
@@ -0,0 +1,36 @@
+// var projects = [];
+
+var url = [NSURL URLWithString:"http://localhost:3000/projects/"];
+
+var request=[NSMutableURLRequest requestWithURL:url cachePolicy:NSURLRequestReloadIgnoringCacheData timeoutInterval:30]
+[request setHTTPMethod:"GET"]
```

Comment:
Just FYI, it's equivalent to `request.setHTTPMethod('GET')', which looks more familiar for us.
http://developer.sketchapp.com/introduction/cocoascript/.